internal/guest/storage/scsi/scsi.go

```diff
@@ -227,10 +225,6 @@ func unmount(
 		trace.Int64Attribute("lun", int64(lun)),
 		trace.StringAttribute("target", target))
 
-	if err = securityPolicy.EnforceDeviceUnmountPolicy(target); err != nil {
```

Comment:
Wasn't this already removed in a PR I did? I removed this. How is this still here? Did something happen with my device unmount PR?

Reply:
yeah, maybe something happened with rebase. I was surprised to see it there too.